.github/actions/upload_terraform_logs/action.yml

```diff
@@ -0,0 +1,11 @@
+name: Upload Terraform logs
+description: "Upload the Terraform log file as an artifact."
+
+runs:
+  using: "composite"
+  steps:
+    - name: Upload artifact
+      uses: actions/upload-artifact@v3
+      with:
+        name: terraform-logs
+        path: terraform.log
```

Comment:
We should add the prefix from the e2e_test as suffix to the filename.

Reply:
Thats what I meant: https://github.com/edgelesssys/constellation/blob/main/.github/actions/e2e_test/action.yml#L170-L177